net/gatling/files/gatling.init

```diff
@@ -0,0 +1,114 @@
+#!/bin/sh /etc/rc.common
+
+# shellcheck shell=ash
+
+# Just looks for changes in the config-file and applies them with a
+# one-time-run.
+
+USE_PROCD=1
+# PROCD_DEBUG=1
+
+# busybox doesn't include the rev-command. Thus implementing it by ourself.
+rev() {
+    while read -r var; do
+        # function taken from https://stackoverflow.com/a/34668251
+        rev=""
+        i=1
+
+        while [ "$i" -le "${#var}" ]; do
+            rev="$(echo "$var" | awk -v i="$i" '{print(substr($0,i,1))}')$rev"
+            : $((i += 1))
+        done
+
+        echo "$rev"
+    done
+}
+
+# taken from /etc/init.d/uhttpd
+append_arg() {
+	local cfg="$1"
+	local var="$2"
+	local opt="$3"
+	local def="$4"
+	local val
+
+	config_get val "$cfg" "$var"
+	[ -n "$val" -o -n "$def" ] && procd_append_param command "$opt" "${val:-$def}"
+}
+
+service_triggers() {
+    procd_add_reload_trigger "gatling"
+}
+
+start_instance() {
+
+    local cfg="$1"
+	local listen http ftp_server
+	local enabled
+
+	config_get_bool enabled "$cfg" 'enabled' 1
+	[ $enabled -gt 0 ] || return
+
+    procd_open_instance
+
+    procd_set_param command /usr/bin/gatling
+    procd_set_param stdout 1
```

Comment:
This will flood the logs during normal usage. How about we comment this out?

Reply:
Yep, will flood the logs. Otherwise, there is a option for that in net/gatling/files/gatling.conf, lines 7 and 21 respectively:
https://github.com/openwrt/packages/blob/fa4b3b36d3ac03fde88bb76b5d49157945ffe495/net/gatling/files/gatling.conf#L7

I would rather prefer setting this to off as default, instead of the need for adjusting the init-script.

Reply:
I've set the option to default off now.